app/api/database-table.js

```diff
@@ -14,6 +14,11 @@ const signedUrl      = require('./signed-url.js');
 const stat           = util.promisify(fs.stat);
 const basePath       = __dirname + '/db-translation-files/';
 
+
+const mkdir         = util.promisify(fs.mkdir);
```

Comment:
You are making promises of these functions but use them synchronously.  Look at fs.writeSync,  fs.statSync .  ect

Reply:
Also if you upgrade nodejs, there is a fspromise api.

Reply:
you can also use all the fs.xyzSync function

Reply:
they are not async

Reply:
I think it's better to _avoid using_ the "Sync" version as it block the main thread. uses of await is better as it release the worker loop to handle other call as the sync function wait for IO

Reply:
yes that the change in the PR